osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -14,11 +15,13 @@ public class GlobalActionContainer : DatabasedKeyBindingContainer<GlobalAction>,
     {
         private readonly Drawable handler;
 
-        public GlobalActionContainer(OsuGameBase game)
+        public GlobalActionContainer(OsuGameBase game, bool nested = false)
```

Comment:
The `nested` param doesn't seem to actually ever be read in any capacity.

Reply:
Remnants of a more messy PoC, can be removed.